pkg/server/commands/listusers/list_users_rpc.go

```diff
@@ -26,6 +26,11 @@ type listUsersQuery struct {
 	resolveNodeBreadthLimit uint32
 }
 
+type foundUser struct {
+	user       *openfgav1.User
+	exceptions []*openfgav1.User
```

Comment:
```suggestion
	excludedUsers []*openfgav1.User
```
To be more consistent with the naming of the API itself.